Cargo.toml

```diff
@@ -16,6 +16,21 @@ members = [
     "examples/container/resource/ferris",
 ]
 
+default-members = [
```

Comment:
This excludes `integration_test` for all other CI checks that do a `cargo check` etc...

Would it be a option to check for the existence of the `north` binary and the test container before starting a test? This is best be done with a proc_macro that wraps `#[tokio::test]`.

Reply:
I was looking for a way to remove the `#[ignore]` from the integration tests and not have them executed when running `cargo test`. One possibility is to _exclude_ the integration tests by changing `cargo test` to `cargo test --lib --bins --examples --benches` which is not that nice.

regarding the issue about checking for `north` and so on, i'm experimenting with https://github.com/crate-ci/escargot to see if i can manage to somehow build the binaries as part of running the test. No much luck so far, but maybe it is possible.